frontend/src/pages/user/seller/info/index.tsx

```diff
@@ -0,0 +1,152 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { Col, Row } from 'react-bootstrap';
+import TButton from '@components/TButton';
+import { useRef, useState } from 'react';
+
+interface ShopInfoProps {
+  name: string;
+  image: File | undefined;
+  enabled: boolean;
+  description: string;
+}
+
+const SellerItemStyle = {
+  borderRadius: '10px',
+  padding: '10% 10% 5% 10%',
+  border: '1px solid var(--button_border, #34977F)',
+  background: 'var(--button_dark, #135142)',
+};
+
+const userImgStyle = {
+  borderRadius: '50%',
+  width: '100px',
+  height: '100px',
+  cursor: 'pointer',
+  boxShadow: '2px 4px 10px 2px rgba(0, 0, 0, 0.25)',
+};
+
+const labelStyle = {
+  fontSize: '16px',
+  fontWeight: '500',
+  color: '#FFFFFFAA',
+};
```

Comment:
color codes typically only have 6 digits perhaps there may be some keyboard errors.

Reply:
that's opacity tho.

Reply:
isn't the last two for alpha values

Reply:
oh i don't know css supports this either
if you want to set opacity with color, i think `rgba(255, 0, 0, 0.2);` is better but it's personal preference

Reply:
it works, but i can change it to rgba(255, 255, 255, 0.67)
![image](https://github.com/jykuo-love-shiritori/twp/assets/91419313/52998ac6-b604-4a84-881f-1bfc547fb580)